protoc-gen-connect-kotlin/src/main/kotlin/com/connectrpc/protocgen/connect/Generator.kt

```diff
@@ -524,3 +539,36 @@ private fun String.packageToDirectory(): String {
     }
     return dir
 }
+
+private fun TypeSpec.Builder.addServiceDeprecation(
+    service: Descriptors.ServiceDescriptor,
+    file: Descriptors.FileDescriptor
+): TypeSpec.Builder {
+    if (service.options.deprecated) {
+        this.addAnnotation(
+            AnnotationSpec.builder(Deprecated::class)
+                .addMember("%S", "The underlying service is marked deprecated.")
```

Comment:
Maybe instead of "underlying" say "Protobuf source file"?
```suggestion
                .addMember("%S", "The service is deprecated in the Protobuf source file.")
```
WDYT?

Reply:
The idea was to match gRPC, but I agree that "underlying" is not clear. Updated in https://github.com/connectrpc/connect-kotlin/pull/334/commits/1a37603b993b3039f6156ddeda439569a3e634c1.